OWNERS

```diff
@@ -7,6 +7,6 @@ approvers:
   - ptabor          # Piotr Tabor <piotr.tabor@gmail.com>
   - spzala          # Sahdev Zala <spzala@us.ibm.com>
 reviewers:
-  - pavelkalinnikov # Pavel Kalinnikov <pavel@cockroachlabs.com>
+  - pav-kv          # Pavel Kalinnikov <pavel@cockroachlabs.com>
```

Comment:
Thanks @pav-kv - Have you changed your github handle or was this never correct?

Reply:
I just changed it, it was correct before.